listenbrainz/labs_api/labs/api/similar_recordings.py

```diff
@@ -18,6 +39,19 @@ def names(self):
         return "similar-recordings", "Similar Recordings Viewer"
 
     def inputs(self):
+        with timescale.engine.begin() as conn:
```

Comment:
We should probably cache this query's result in redis with a TTL of an hour or so.